week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -59,6 +75,21 @@ function exerciseTwo(shopping) {
 **/
 function exerciseThree(books) {
   //Write your code in here
+  
+  let unord = document.createElement("ul");
+  document.body.appendChild(unord);
+  books.forEach(book => {
+    let pEls = document.createElement("p");
+    pEls.innerHTML = `${book.title}, ${book.author}`;
+    console.log(pEls);
+    let listEls = document.createElement("li");
```

Comment:
- you lift the images empty. you can do this by using `imgEls.src=' path/to/image '`
- you did not remove the bullets from ul. you can do this by using css property: 
```css
ul {
       list-style-type: none;
}
```
- you can use css: `width: 25%;`  to control each book container.
- you can use css: `float:left;` to align the book containers horizontally , You might find other options for this one.
- compared with this file: https://hyf-js2-week1-makeme-ex1-demo.herokuapp.com/index.html 

- resources:
list style : https://www.w3schools.com/css/css_list.asp
float: https://www.w3schools.com/css/css_float.asp
another float: https://developer.mozilla.org/en-US/docs/Web/CSS/float

Reply:
You are right. I didn't style the page.